src/phoenix/db/bulk_inserter.py

```diff
@@ -108,27 +113,25 @@ async def _insert_evaluations(self) -> None:
                         try:
                             async with session.begin_nested():
                                 await _insert_evaluation(session, evaluation)
-                        except Exception:
-                            logger.exception(
-                                "Failed to insert evaluation "
-                                f"for span_id={evaluation.SubjectId.span_id}"
-                            )
+                        except InsertEvaluationError as error:
+                            logger.exception(f"Failed to insert evaluation: {str(error)}")
```

Comment:
i think this printing is automatic, so it would be a duplication
```suggestion
                        except InsertEvaluationError:
                            logger.exception(f"Failed to insert evaluation")
```

Reply:
The more granular error message is included in the stacktrace but not the log messages. I think it's worth keeping.